tests/console/consoletest_finish.pm

```diff
@@ -37,12 +37,14 @@ sub run() {
 
     save_screenshot();
 
-    if (check_var("DESKTOP", "textmode")) {
+    # we don't have a tty1 on s390 and just stay on the same console
+    # (see consoletest_setup)
+    if (check_var("DESKTOP", "textmode") && !check_var('ARCH', 's390x')) {
         send_key "ctrl-alt-f1";    # go back to first console
         assert_screen 'linux-login', 10;
     }
     else {
-        send_key "ctrl-alt-f7";    # go back to X11
+        select_console('x11');     # go back to X11
```

Comment:
most likely - yes...  
we never got this far with textmode but it would have failed anyway.
but I think this needs more testing tomorrow then ;)